### PR TITLE
Enhanced process for managing gem resources 

### PR DESCRIPTION
**Pull request by RailsCod3rFuture**
It was decided that gem resources should live in their own respective domain and we should remove the Gemfile.lock. So that AWS Inspector will not attempt to signal vulnerabilities exist -- that don't really.

### CODE REVIEW

**Review by tomdracz — CHANGES_REQUESTED** [review]
One thing missing - we should gitignore the Gemfile.lock and this should be good to go!

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /.bundle
+/Gemfile.lock
```
Comment (tomdracz) [inline .gitignore:2]: Just small tweak, let's lose the trailing slash so it's clear we're referencing file and not directory

**Review by tomdracz — CHANGES_REQUESTED** [review]
One small change and it's good to go

File: .gitignore (hunk 1 above)
Comment (RailsCod3rFuture) [reply .gitignore]: made your suggestion.

**Review by tomdracz — APPROVED** [review]

**Pull request merged**